### DISCUSSION TITLE
should we migrate UI from react router v5 to v6

### DISCUSSION THREAD

**Discussion by Aman-Singh-Kushwaha** [superstreamlabs/memphis#1348] [Ideas]
React Router v6 comes with a major update in version 6, like 
Improved performance for loading faster ,Enhanced security, Greater flexibility.

@idanasulinmemphis @Avitaltrifsik

**Comment by Aman-Singh-Kushwaha** [comment]
If it works, i would happy to do

**Reply by yanivbh1** [reply]
Hey @Aman-Singh-Kushwaha ,
Great to have you here!
@avrhamNeeman what do you think?

**Reply by avrhamNeeman** [reply]
@Aman-Singh-Kushwaha, Thank you for bringing up this issue.
It would be great if you could start working on it.
Please ensure that this migration has not caused any issues in the existing code.

**Reply by Aman-Singh-Kushwaha** [reply]
Thanks for your kind approval. I had started issue #1349  to track progress for PR.
